exercises/edward_brown/ex01_basics/src/Punctuation.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Punctuation.h"
+#include <algorithm>
+
+bool isPunctuation(char Candidate) {
+  auto const punctuation = std::string(".,?'\\\"!():");
```

Comment:
Good points :+1: I'm not convinced by point 3 but maybe I'm just old-fashioned ;) It looks like this style is used a bit in Mantid although not that much, so the rule of thumb is to fit in with the local style where appropriate.